test/metric_value_benchmark/statsd_test.go

```diff
@@ -6,62 +6,25 @@
 package metric_value_benchmark
 
 import (
-	"fmt"
 	"log"
 	"time"
 
+	"github.com/aws/aws-sdk-go-v2/aws"
+
+	"github.com/aws/amazon-cloudwatch-agent-test/internal/common"
 	"github.com/aws/amazon-cloudwatch-agent-test/test/metric"
 	"github.com/aws/amazon-cloudwatch-agent-test/test/metric/dimension"
 	"github.com/aws/amazon-cloudwatch-agent-test/test/status"
 	"github.com/aws/amazon-cloudwatch-agent-test/test/test_runner"
-	"github.com/aws/aws-sdk-go-v2/aws"
-	"github.com/cactus/go-statsd-client/v5/statsd"
 )
 
 var _ test_runner.ITestRunner = (*StatsdTestRunner)(nil)
+
 type StatsdTestRunner struct {
 	test_runner.BaseTestRunner
 }
 
-// stopChan is for stopping the goroutine generating statsd metrics.
```

Comment:
Deleting this since the `metric_value_benchmark` are sharing the resources to each other. Therefore, the resources need to be unique within each file. Instead of generate statsd metrics here, use a common function [sendStatdMetrics](https://github.com/aws/amazon-cloudwatch-agent-test/blob/main/internal/common/metrics.go#L86-L118) to send the required metrics. The long term would replace the metric [value benchmark with the validator](https://github.com/aws/amazon-cloudwatch-agent-test/tree/main/validator)